File: Desktop/Domain/CmdMsgs/LevelCmd.cs
using System;

using System.Xml;
using System.Xml.Xsl;
using System.Collections;
using System.Runtime.InteropServices;

namespace Blurts
{
  [ComVisible(true)]
  public class LevelCmd : CmdBase
  {
    // Tag Names

    public LevelCmd(String s)
      : base(s)
    {
    }

    public LevelCmd()
      : base(CMD_LEVEL)
    {
    }
  }
}

File: BlackBerry/AlertMsgs/LevelCmd.java
package com.mlhsoftware.Blurts;

import org.json.me.*;


class LevelCmd extends CmdBase 
{
  // Tag Names


  public LevelCmd( String string ) throws JSONException
  {
    super( string );
  }

  public LevelCmd()
  {
    super( CMD_LEVEL );
  }
}
